### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by rossturk**
A series of quick fixes to our new website:

- hid cookie banner that linked to a nonexistent cookie policy
- google analytics tracking
- added author field to blogs
- added a favicon

### CODE REVIEW

**Review by wslulciuc — APPROVED** [review]

**Comment by rossturk** [comment]
Sorry, I snuck a few more commits in there after your review @wslulciuc :) I added support for SVG banners and then stumbled into two other things I wanted to fix.

**Comment by wslulciuc** [comment]
Ahh, no worries. New changes / fixes look great!

File: src/components/item-blog.tsx [hunk 1]
```diff
@@ -16,19 +16,17 @@ export const ItemBlog: React.FC<{ data: ItemBlogProps}> = ({ data }) => {
...
                 <Link to={data.fields.slug} title={data.frontmatter.title} onFocus={() => changeFocused(true)} onBlur={() => changeFocused(false)}>
                     <div className="image">
-                        <Img
-                            fluid={data.frontmatter.image.childImageSharp.fluid}
-                            alt={data.frontmatter.title}
-                            className="w-full"
-                        />
+                        {data.frontmatter.image.publicURL.endsWith('.svg') ?
+                        <img src={data.frontmatter.image.publicURL} alt={data.frontmatter.title} className="w-full" /> :
+                        <Img fluid={data.frontmatter.image.childImageSharp?.fluid} alt={data.frontmatter.title} className="w-full" />}
```
Comment (phixMe) [inline src/components/item-blog.tsx:21]: Might be worth adding prettier to the project as a next task. 
https://prettier.io/

File: src/templates/blog.tsx [hunk 2]
```diff
@@ -23,11 +23,9 @@ export default function blog({ location, data }: PageProps<BlogQuery, {}>) {
...
             <div className="md:px-4 mt-12 py-6 md:w-11/12 mx-auto">
                 <div className="mx-auto relative">
-                    <Img
-                        fluid={
-                            data.mdx.frontmatter.banner.childImageSharp.fluid
-                        }
-                    />
+                    {data.mdx.frontmatter.banner.publicURL.endsWith('.svg') ?
+                    <img src={data.mdx.frontmatter.banner.publicURL} alt="''"/> :
+                    <Img fluid={data.mdx.frontmatter.banner.childImageSharp.fluid}/>}
```
Comment (phixMe) [inline src/templates/blog.tsx:28]: We can build this as a component to proxy this logic in the general sense.

**Review by phixMe — APPROVED** [review]
Looks good, thanks @rossturk

File: src/templates/blog.tsx (hunk 2 above)
Comment (rossturk) [reply src/templates/blog.tsx:28]: I think that would be awesome!

File: src/components/item-blog.tsx (hunk 1 above)
Comment (rossturk) [reply src/components/item-blog.tsx:21]: Is that a replacement for the Prism plugin we're using now?

Comment (phixMe) [reply src/components/item-blog.tsx:21]: I would standardize our code formatting across all the files to make this indentation more normalized.

**Pull request merged**